src/renderer/services/auto-updater.service.ts

```diff
@@ -44,6 +56,34 @@ export class AutoUpdaterService{
         this.ipcService.sendLazy("install-update");
     }
 
+    public getHaveBeenUpdated() : Observable<boolean>{
+        return this.ipcService.sendV2<boolean>("have-been-updated");
+    }
+
+  public async getChangelogs(): Promise<Changelog | null> {
+    try {
+      const path = `https://raw.githubusercontent.com/Zagrios/bs-manager/master/assets/jsons/changelogs/${this.i18nService.currentLanguage.split("-")[0]}.json`;
+
+      const response = await fetch(path);
+      if (!response.ok) {
+        return;
+      }
+
+      const data = await response.json();
+      if (!data?.body) {
+        return;
+      }
+      return data;
+
+    } catch (error) {
+      return;
```

Comment:
no need return here, you can use the thing at line 15 in index.tsx to log the error

Reply:
it's realy useless ? 
if i remove the return, have a error "Not all code paths return a value." 👀

Reply:
catch (error) {
      this.ipcService.sendLazy("log-error", { args: error });
    }

Reply:
```suggestion
  } catch (error) {
      this.ipcService.sendLazy("log-error", { args: error });
    }
```